dev/com.ibm.ws.classloading/resources/com/ibm/ws/classloading/internal/resources/ClassLoadingServiceMessages.nlsprops

```diff
@@ -104,6 +104,10 @@ cls.library.path.invalid=CWWKL0019W: The [{0}] library specifies a path with the
 cls.library.path.invalid.explanation=To add a path to a library in the server configuration, you must specify a path that exists.
 cls.library.path.invalid.useraction=Edit the server configuration and modify the path attribute of the library element.
 
+app.classloader.removed=CWWKL0020W: The thread context class loader for thread [{0}] no longer references stopped application [{1}].
```

Comment:
change
for thread [{0}] no longer references stopped application [{1}].
to
for the [{0}] thread no longer references the [{1}] application, which is stopped.